apps/fabric-website/index.html

```diff
@@ -1,157 +1,161 @@
-    }
-
-    // Simple synchronous script loader with callback. Adapted from https://stackoverflow.com/questions/1866717
-    function loadScripts(array, callback) {
-      var loader = function (src, handler) {
-        var script = document.createElement('script');
-        script.src = src;
-        script.onload = script.onreadystatechange = function () {
-          script.onreadystatechange = script.onload = null;
-          handler();
-        }
-        var head = document.getElementsByTagName('head')[0];
-        (head || document.body).appendChild(script);
-      };
-      (function run() {
-        if (array.length != 0) {
-          loader(array.shift(), run);
-        } else {
-          callback && callback();
-        }
-      })();
-    }
+        var configScript = ['https://fabricweb.azureedge.net/fabric-website/manifests/' + configPrefix + '.js'];
 
-    function loadAppScripts(appPath) {
-      var reactPath = '//cdnjs.cloudflare.com/ajax/libs/react/16.3.2/umd/react';
-      var reactDomPath = '//cdnjs.cloudflare.com/ajax/libs/react-dom/16.3.2/umd/react-dom';
+        loadScripts(configScript, function() {
+          if (window.Flight) {
+            appPath += window.Flight.baseCDNUrl;
 
-      if (noCompress) {
-        reactPath = reactPath + '.development.js';
-        reactDomPath = reactDomPath + '.development.js';
-        appPath += entryPointFilename + '.js';
+            loadAppScripts(appPath);
+          }
+        });
       } else {
+        loadAppScripts(appPath);
+      }
+
+      // Simple synchronous script loader with callback. Adapted from https://stackoverflow.com/questions/1866717
+      function loadScripts(array, callback) {
+        var loader = function(src, handler, failHandler) {
+          var script = document.createElement('script');
+          script.src = src;
+          script.onload = script.onreadystatechange = function() {
+            script.onreadystatechange = script.onload = null;
+            handler();
+          };
+
+          script.onerror = script.onreadystatechange = function() {
+            script.onreadystatechange = script.onerror = null;
+            if (failHandler) {
+              failHandler();
+            }
+          };
+
+          var head = document.getElementsByTagName('head')[0];
+          (head || document.body).appendChild(script);
+        };
+        (function run() {
+          if (array.length != 0) {
+            const scriptName = array.shift();
+            loader(scriptName, run, () => {
+              loader(scriptName.replace('.min.js', '.js'), run);
+            });
+          } else {
+            callback && callback();
+          }
+        })();
+      }
+
+      function loadAppScripts(appPath) {
+        var reactPath = '//cdnjs.cloudflare.com/ajax/libs/react/16.3.2/umd/react';
+        var reactDomPath = '//cdnjs.cloudflare.com/ajax/libs/react-dom/16.3.2/umd/react-dom';
+
         reactPath = reactPath + '.production.min.js';
```

Comment:
Is it intentional to remove the ability to run with development React builds?

Reply:
Yup - we shouldn't debug into react builds really. Unless we're really into looking at Fiber code (which I don't think we ever should do). We can put this back on if we really it back someday. Otherwise, this simplifies the retry with non-min logic